package.json

```diff
@@ -33,10 +33,10 @@
     "@material-ui/icons": "^4.5.1",
     "@material-ui/styles": "^4.5.0",
     "axios": "^0.19.0",
-    "node-sass": "^4.12.0",
     "prop-types": "^15.7.2",
     "react": "^16.10.2",
     "react-dom": "^16.10.2",
+    "react-router": "^5.1.2",
```

Comment:
You are changing style, but why add this ```react-router```

Reply:
I was building Storybook onto production and error pops up saying it requires `react-router`